cmd/version.go

```diff
@@ -20,11 +26,60 @@ var versionCmd = &cobra.Command{
 To see if there is a more recent version available, call the command with the
 --latest flag.
 	`,
+
 	Run: func(cmd *cobra.Command, args []string) {
```

Comment:
I agree named functions read better. But my reason for changing from a named function into an anonymous function was to prevent the extra lookup (someone reading the code) and to prevent it from being called on its own. The function itself is nothing more than a wrapper around the two functions `currentVersion and checkForUpdate` so I thought it was best to define within versionCmd. 

But if it is more readable as a named function I can easily break it out.